### ISSUE TITLE
Bravo!! Is it ready for production environment ?

### ISSUE THREAD

**Issue by huangzhhui** [zendframework/zend-expressive-swoole#6]
Bravo!! Is it ready for production environment ? I would like to use it !

**Comment by ezimuel** [comment]
@huangzhhui Thanks! Not yet, we are working on it and we'll release asap.

**Comment by huangzhhui** [comment]
@ezimuel Have any todo list or bug list about this project ? I will try to complete some issue, I am the heavy user of Zend Framework, And I am the author of [Swoft](https://github.com/swoft-cloud/swoft) ( Most popular coroutine framework base on Swoole ), I have experience with this

**Comment by ezimuel** [comment]
@huangzhhui thanks a ton for your help! I provided a short *To Do* list for the Swoole integration [here](https://github.com/zendframework/zend-expressive-swoole/projects/1).

**Comment by huangzhhui** [comment]
@ezimuel I notice that the issue in [Project](https://github.com/zendframework/zend-expressive-swoole/projects/1) rely on your un-merged pr https://github.com/zendframework/zend-expressive-swoole/pull/5 , I would like to know when you might merge this pr.

**Comment by ezimuel** [comment]
@huangzhhui very soon, waiting for @weierophinney review.

**Comment by ezimuel** [comment]
@huangzhhui maybe this can be interesting for you https://github.com/ezimuel/swoole-benchmark

**Comment by huangzhhui** [comment]
@ezimuel The results of your test are interesting, but usually we use Nginx for static files, cuz HTTP Protocol Support of Swoole is not well, https and e-tag etc.

**Comment by weierophinney** [comment]
@huangzhhui You're absolutely correct - we should likely allow users to define a suite of headers to use by default with static files as well. I've created #10 to track it.

**Comment by huangzhhui** [comment]
@weierophinney @ezimuel `sendfile` also does not support `gzip`, it is not really good for static file transport.

**Comment by ezimuel** [comment]
@huangzhhui I know that sendfile does not support gzip, but we can implement it using the [Zlib functions](http://php.net/manual/en/ref.zlib.php) of PHP.
